diff --git a/node_modules/ai-visual-tester/index.js b/node_modules/ai-visual-tester/index.js
index 2a51e70..977c33d 100644
--- a/node_modules/ai-visual-tester/index.js
+++ b/node_modules/ai-visual-tester/index.js
@@ -27,11 +27,11 @@ async function testPhoto(file) {
 
     const headers = {
         "Content-Type": "application/json",
-        "Authorization": `Bearer ${process.env.OPENAI_KEY}`
+        "Authorization": `Bearer ${process.env.OPENAI_API_KEY}`
     };
 
     const payload = {
-        "model": "gpt-4-vision-preview",
+        "model": "gpt-4o",
         "messages": [
             {
                 "role": "user",
